data/theme/gnome-shell.css

```diff
@@ -2166,8 +2168,17 @@ stage {
     font-size: 9pt;
 }
 
+.aggregate-menu-indicators-box {
+    padding: 6px 4px;
+    background-color: rgba(255, 255, 255, 0.1);
+    border-radius: 3px;
+}
+
 #panel .endless-button,
 #panel.solid .endless-button {
+    -natural-hpadding: 12px;
+    -minimum-hpadding: 6px;
```

Comment:
I understand these changes are required here to explicitly state a different padding for the Endless button compared to the icons in the right side of the taskbar, however... 

Please consider either doing it the other way around (leaving the default padding for ` #panel .panel-button` -see my previous comment- unchanged + explicitly setting it to 10px for the right side of the panel), or doing it this way but then splitting the changes to `#panel.solid .endless-button` to a separate commit, so that we can squash it along with other Endless button-related changes in future rebases.

Either way, please do not modify the CSS above the `/* Endless-specific overrides below this point */` comment above.